prisma/schema.prisma

```diff
@@ -200,7 +203,51 @@ model Invitation {
   createdAt DateTime @default(now())
   token     String   @unique
 
-  @@unique([email,teamId])
+  @@unique([email, teamId])
+}
+
+model Webhook {
+  id        String   @id @default(cuid())
+  userId    String
+  user      User     @relation(fields: [userId], references: [id], onDelete: Cascade)
+  targetUrl String
+  events    Event[]
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+}
+
+model Notification {
+  id         String   @id @default(cuid())
+  userId     String
+  user       User     @relation(fields: [userId], references: [id], onDelete: Cascade)
```

Comment:
Notifications should be also scoped by `team` so that i'm receiving the proper notifications for the team i'm part of